website/cue/reference/components/transforms/base/log_to_metric.cue

```diff
@@ -55,8 +55,12 @@ base: components: transforms: log_to_metric: configuration: metrics: {
 			description: "Tags to apply to the metric."
 			required:    false
 			type: object: options: "*": {
-				description: "Tags to apply to the metric."
-				required:    true
+				description: """
+					Specification of the value of a created tag.
+
+					This may be a single value, a `null` for a bare tag, or an array of either.
+					"""
+				required: true
```

Comment:
Are bare/multi tags completely supported already?

Reply:
Well, technically yes, since the code is merged in.

Reply:
Looking a little closer, though... I did just realize that we're not generating a completely valid JSON Schema for the `TagConfig` type, as it doesn't actually allow for a `null` value to describe a bare tag.

Gonna either see if I can trivially fix that here, and make an issue for it to fix it as the very next thing.

Reply:
👍 wasn't entirely sure if this should be hidden in the documentation yet, as it's not implemented end-to-end

Reply:
Yeah, I mean, given how and where it's exposed, it's supported in terms of the `log_to_metric` transform, but there's not really a way for us to hide that at the moment.